frontend/src/framework/HoverService.ts

```diff
@@ -0,0 +1,195 @@
+import React from "react";
+
+import { throttle } from "lodash";
+
+import { PublishSubscribeDelegate } from "@lib/utils/PublishSubscribeDelegate";
+
+const HOVER_UPDATE_THROTTLE_MS = 100;
+
+export enum HoverTopic {
+    WELLBORE_MD = "hover.md",
+    WELLBORE = "hover.wellbore",
+    REALIZATION = "hover.realization",
+    TIMESTAMP = "hover.timestamp",
+    ZONE = "hover.zone",
+    REGION = "hover.region",
+    FACIES = "hover.facies",
+    WORLD_POS = "hover.world_pos",
```

Comment:
I think it would be good to clarify which type off coordinates we expect - e.g. utm-coordinates (i assume). Either do it in the naming of hover topic, or in the data below, i.e. `x` renamed to `xUtm`, and same for `y`. z is usually without suffix utm i believe.